완주하지-못한-선수.js

```diff
@@ -0,0 +1,15 @@
+function solution(participant, completion) {
+    const runners = {};
+
+    for (let p of participant) {
+    runners[p] = runners[p] ? runners[p] + 1 : 1;
+       //console.log(p)
+    }
+    for (let c of completion) {
+    runners[c] -= 1;
+    }
+    for (let key in runners){
+        if (runners[key] === 1)
+        return key;
```

Comment:
중괄호 없이 조건문을 실행시켜 줄땐 한줄로 작성해주세요!
```suggestion
        if (runners[key] === 1) return key;
```